source/Ship.cpp

```diff
@@ -47,11 +48,11 @@ namespace {
 	const vector<string> BAY_TYPE = {"drone", "fighter"};
 	const vector<string> BAY_SIDE = {"inside", "over", "under"};
 	const vector<string> BAY_FACING = {"forward", "left", "right", "back"};
-	const vector<Angle> BAY_ANGLE = {Angle(0.), Angle(-90.), Angle(90.), Angle(180.)};
+	const vector<Angle> BAY_ANGLE = {Angle(DEG_0), Angle(-DEG_90), Angle(DEG_90), Angle(DEG_180)};
```

Comment:
Shouldn't `-DEG_90` be `DEG_270`?

Reply:
Yes, of course, they are geometrically equivalent but, considering they are relative to an oriented system (the ship vector), I think representing left as counterclockwise angle is more expressive -- for example, I saw the same notation in OpenCV:

```
enum RotateFlags {
    ROTATE_90_CLOCKWISE = 0, //!<Rotate 90 degrees clockwise
    ROTATE_180 = 1, //!<Rotate 180 degrees clockwise
    ROTATE_90_COUNTERCLOCKWISE = 2, //!<Rotate 270 degrees clockwise
};```

Reply:
Yeah, for rotations a [-180°, 180°) range makes sense, for positions a [0, 360°) range. I vaguely recall reading a comment to that effect somewhere.
Anyway, I don't really care either way, both options are clear enough for me (as is `-90.`).